app/controllers/patients_controller.rb

```diff
@@ -89,6 +89,18 @@ def data_entry_create
     end
   end
 
+  # allow ONLY admins to delete patients, and only then if patient does not have pledge_sent
+  def destroy
+    @patient = Patient.find params[:patient_id]
+    if current_user.admin? && !@patient.pledge_sent?
+      current_user.remove_patient(@patient)
```

Comment:
Rather than using `remove_patient`, let's use `@patient.destroy` so we can get this out of db. (Noted below, but `remove_patient` is totally an example of us naming something badly a year and a half ago, sorry about that! Def my fault)